ruby/rollbar.rb

```diff
@@ -0,0 +1,10 @@
+Rollbar.configure do |config|
+  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
```

Comment:
Dat de app dan helemaal niet start is precies het punt, lijkt me: als de waarde er niet is, heeft een developer (waarschijnlijk jijzelf) het verkloot en moet je er direct mee kappen. Als je dat niet doet, kom je er waarschijnlijk pas veel later achter dat er iets niet werkt zoals je had verwacht.

Dit is geen user input die er soms wel, soms niet is. Dit is gewoon “code” die er altijd moet zijn.